ChaosMod/Memory/Hooks/DrawingHooks.cpp

```diff
@@ -0,0 +1,76 @@
+#include <stdafx.h>
+
+#include "Hook.h"
+#include "Util/Color.h"
+
+// Thanks CitizenFX!
+// Adapted by MoneyWasted for ChaosModV
+// https://github.com/citizenfx/fivem/blob/master/code/components/extra-natives-five/src/Draw2dNatives.cpp
+
+// Notes:
+// Analysis in IDA shows that the return values of
+// _HK_allocateDrawRect is most likely a structure
+// of some sorts.
+
+static char *g_drawRects           = 0;
+static int *g_mainThreadFrameIndex = 0;
+static int g_drawRectsSize         = 0;
```

Comment:
```suggestion
static char *ms_DrawRects           = 0;
static int *ms_MainThreadFrameIndex = 0;
static int ms_DrawRectsSize         = 0;
```
(Of course has to be adapted in other places as well)